wiki/How-to-use-the-Docker-image.md

```diff
@@ -1,18 +1,26 @@
 # How to launch your bot
+## Download the docker image
+The images are available through [Docker hub](https://hub.docker.com/r/lichessbotdevs/lichess-bot) and [Github container registry](https://github.com/lichess-bot-devs/lichess-bot/pkgs/container/lichess-bot).
+Run `docker pull lichessbotdevs/lichess-bot` to download from docker hub, or run `docker pull ghcr.io/fathzer/lichess-bot` if you want to download it from the Github container registry.
```

Comment:
Is `docker pull ghcr.io/fathzer/lichess-bot` correct?

Reply:
Fixed.